public/actions/CampaignActions/getCampaignContent.js

```diff
@@ -0,0 +1,37 @@
+import {fetchCampaignContent} from '../../services/CampaignsApi';
+
+function requestCampaignContent(id) {
+   return {
+        type:       'CONTENT_GET_REQUEST',
+        id:         id,
+        receivedAt: Date.now()
+    };
+}
+
+function recieveCampaignContent(content) {
```

Comment:
turns out all the actions have the same typo, so I'm going to put this in as a tech debt ticket and not fix here.